tests/integration/wordpress_client.py

```diff
@@ -108,7 +108,7 @@ def _gen_wp_rest_nonce(self):
         nonce = json.loads(re.findall('var wpApiSettings = ([^;]+);', new_post_page)[0])["nonce"]
         return nonce
 
-    def _create_post(self, title: str, content: str):
+    def create_post(self, title: str, content: str):
         """Create a WordPress post using wp-json API, return post object"""
```

Comment:
Do you think it is worth updating the docstring to our new standard and to define a return type?

Reply:
Fixed, thanks!